File: Hexaware/python/guessTheWord.py
def guessTheWord(n, string):
    stringsArr = string.split(" ")
    length = 0
    index = 1
    updated = False
    print(stringsArr)
    for word in stringsArr:
        if len(word) % 2 != 0 and len(word) > length:
            length = len(word)
            index = stringsArr.index(word)
            updated = True
    if (updated):
        return stringsArr[index]
    else:
        return "Better Luck Next Time"


print(guessTheWord(5, "Helo Good Mornin Welcum Yu"))

File: Hexaware/javascript/guessTheWord.js
function guessTheWord(n, string){
    let stringsArr = string.split(" ")
    let index = 0;
    let length = -Infinity;
    let updated = false;
    for (const word of stringsArr){
        if (word.length % 2 !== 0){
            if (length < word.length){
                length = word.length
                index = stringsArr.indexOf(word);
                updated = true;
            };
        }
    }
    if (!updated) return "Better Luck Next Time"
    else return stringsArr[index];
}


console.log(guessTheWord(5, "Helo Good Mornin Welcum Yu"))